### PR TITLE
server up and running with two unit tests

### PR DESCRIPTION
**Pull request by leungnamson1023**
https://trello.com/c/0TWJVjEp/20-get-server-up-and-running-with-two-unit-tests-that-test-a-get-route-for-a-static-file-should-200-if-correct-url-404-if-otherwise

### CODE REVIEW

File: spec/server.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const server = require('../server/app.js');
```
Comment (tmkuba) [inline spec/server.test.js]: maybe rename this variable to app so that it isn't confused with server/server.js

**Review by tmkuba — COMMENTED** [review]
minor variable name change suggestion (not required)

**Comment by leungnamson1023** [comment]
@tmkuba i changed the variable name for consistency, review whenever you have a chance

**Review by tmkuba — COMMENTED** [review]
looks good!

**Pull request merged**